instrumentation/log4j/log4j-appender-2.16/library/README.md

```diff
@@ -0,0 +1,77 @@
+# Log4j2 Appender
+
+This module provides a Log4j2 [appender](https://logging.apache.org/log4j/2.x/manual/appenders.html)
+which forwards Log4j2 log events to
+the [OpenTelemetry Log SDK](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk/logs)
+.
+
+To use it, add the following modules to your application's classpath.
+
+Replace `OPENTELEMETRY_VERSION` with the latest
+stable [release](https://search.maven.org/search?q=g:io.opentelemetry).
+
+**Maven**
+
+```xml
+<dependencies>
+  <dependency>
+    <groupId>io.opentelemetry.instrumentation</groupId>
+    <artifactId>opentelemetry-log4j-appender-2.16</artifactId>
+    <version>OPENTELEMETRY_VERSION</version>
+    <scope>runtime</scope>
+  </dependency>
+  <dependency>
+    <!-- The SDK appender is required to configure the appender with the OpenTelemetry Log SDK -->
+    <groupId>io.opentelemetry.instrumentation</groupId>
+    <artifactId>opentelemetry-instrumentation-sdk-appender</artifactId>
+    <version>OPENTELEMETRY_VERSION</version>
+  </dependency>
+</dependencies>
+```
+
+**Gradle**
+
+```kotlin
+dependencies {
+  runtimeOnly("io.opentelemetry.instrumentation:opentelemetry-log4j-appender-2.16:OPENTELEMETRY_VERSION")
+  // The SDK appender is required to configure the appender with the OpenTelemetry Log SDK
+  implementation("io.opentelemetry.instrumentation:opentelemetry-instrumentation-sdk-appender:OPENTELEMETRY_VERSION")
+}
+```
+
+The following demonstrates how you might configure the appender in your `log4j2.xml` configuration:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN" packages="io.opentelemetry.instrumentation.log4j.appender.v2_16">
+  <Appenders>
+    <Console name="Console" target="SYSTEM_OUT">
+      <PatternLayout
+          pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} traceId: %X{trace_id} spanId: %X{span_id} flags: %X{trace_flags} - %msg%n"/>
+    </Console>
+    <OpenTelemetry name="OpenTelemetryAppender"/>
+  </Appenders>
+  <Loggers>
+    <Root>
+      <AppenderRef ref="OpenTelemetryAppender" level="All"/>
+      <AppenderRef ref="Console" level="All"/>
+    </Root>
+  </Loggers>
+</Configuration>
+```
+
+Next, associate the `OpenTelemetryAppender` with a `SdkLogEmitterProvider` in your application:
```

Comment:
I don't see any `OpenTelemetryAppender` in the example below. Maybe I don't know anything about log4j appenders, but I am confused here :)

Reply:
`OpenTelemetryAppender` is the name given to the appender configured in the xml snippet [above](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/4985/files#diff-4e8db946763ef719aaa67fb488575bc315fb1770b6778c730b9278199f628aebR52-R56).

Reply:
Yes, but this name is not present in the code. So how exactly do I "associate the `OpenTelemetryAppender` configured via `log4j2.xml` with a `SdkLogEmitterProvider`"? What if I use another name in log4j.xml, do I need to change my code somehow?